mono/metadata/handle.h

```diff
@@ -614,24 +606,12 @@ mono_gchandle_from_handle (MonoObjectHandle handle, mono_bool pinned);
 MonoObjectHandle
 mono_gchandle_get_target_handle (uint32_t gchandle);
 
-static inline gboolean
-mono_gchandle_target_equal (uint32_t gchandle, MonoObjectHandle equal)
-{
-	// This function serves to reduce coop handle creation.
-	MONO_HANDLE_SUPPRESS_SCOPE (1);
-	return mono_gchandle_get_target (gchandle) == MONO_HANDLE_RAW (equal);
-}
+gboolean
+mono_gchandle_target_equal (uint32_t gchandle, MonoObjectHandle equal);
```

Comment:
Why? this isn't a public header

Reply:
You can just update this inline function (and the others in handle.h to call the _internal functions.

Reply:
Cannot include object-internals.h without causing more breaks, and didn't work them through.

Reply:
Ah ok.  gotcha.